.github/workflows/pod_failover_test.yml

```diff
@@ -0,0 +1,81 @@
+name: Pod Failover Test
+on:
+    schedule:
+    - cron: "0 13 * * *"
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    env:
+      AZURE_TENANT_ID: ${{ secrets.TenantId }}
+      AZURE_SUBSCRIPTION_ID: ${{ secrets.SubscriptionId }}
+      AZURE_CLIENT_ID: ${{ secrets.ClientId }}
+      AZURE_CLIENT_SECRET: ${{ secrets.ClientSecret }}
+      AZURE_LOCATION: ${{ secrets.E2ELocation }}
+      REGISTRY: e2etest.azurecr.io/mandalay
+      METRICS_ENDPOINT: ${{ secrets.MetricsEndpoint }}
```

Comment:
This implies we have a publicly accessible endpoint to push the metrics to. How is this being secured? I am concerned that we may be flagged for S360 violations here.